recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ requirements:
 
   run:
     - python
-    - numpy x.x
+    - numpy >=1.11
     - jpeg 9.*
```

Comment:
Also please drop the `.` here.

Reply:
For this one, I'm not sure what you mean.